CMakeLists.txt

```diff
@@ -234,6 +234,13 @@ option(WITH_KVS "Key value store is here" ON)
 # remote block storage
 option(WITH_RBD "Remote block storage is here" ON)
 
+# embedded ceph daemon static library
+# NOTE: Ceph is mostly LGPL (see COPYING), which means that
+# static linking brings with it restrictions. Please be sure
+# to look at the LGPL license carefully before linking this library to
+# your code. See http://www.gnu.org/licenses/gpl-faq.html#LGPLStaticVsDynamic.
+option(WITH_EMBEDDED "build the embedded ceph daemon library" ON)
```

Comment:
@bassamtabbara 

if the sole purpose of enabling this option by default is to avoid the code from rotting, maybe we can at least disable it in `debian/rules` and `ceph.spec.in`.